extensions/import/src/wizard/pages/fileConfigPage.ts

```diff
@@ -340,7 +340,14 @@ export class FileConfigPage extends ImportPage {
 		let connectionUri = await sqlops.connection.getUriForConnection(this.model.server.connectionId);
 		let queryProvider = sqlops.dataprotocol.getProvider<sqlops.QueryProvider>(this.model.server.providerName, sqlops.DataProviderType.QueryProvider);
 
-		let query = `SELECT name FROM sys.schemas`;
+		let query = '';
+		if (this.databaseDropdown.value != undefined)
+		{
+			query = `SELECT name FROM [${this.databaseDropdown.value}].sys.schemas`;
```

Comment:
This throws an error when I run it on my computer because `this.databaseDropdown.value` is an object with "name" and "displayName" attributes, so I think this needs to be `this.databaseDropdown.value.name`.

Reply:
When I add .name here is what I get...

![image](https://user-images.githubusercontent.com/17560259/46879847-1793d700-ce15-11e8-9868-8b41332946b8.png)